cobbler/actions/reposync.py

```diff
@@ -27,12 +27,7 @@
 import os.path
 import pipes
 import stat
-
-HAS_YUM = True
-try:
-    import yum
-except:
-    HAS_YUM = False
+import shutil
 
 HAS_LIBREPO = True
 try:
```

Comment:
We need to eliminate the `try..except` block for librepo because we no longer have an alternate codepath that works. This also means that `python3-librepo` is a hard dependency instead of a weak one.

Reply:
If I now remove the try..except block, then in this place there will be an error in RedHat/CentOS 7 when checking. Is there a need to support version 7 of distributions?

Reply:
It is also possible that there is no librepo in Debian/Ubuntu. Then you can use rsync without causing an error at this point.